wasm/src/lib.rs

```diff
@@ -141,7 +141,17 @@ pub fn get_circuit(
         .circuit(entry_point, simulate)
         .map_err(interpret_errors_into_qsharp_errors_json)?;
 
-    serde_wasm_bindgen::to_value(&circuit).map_err(|e| e.to_string())
+    let circuitData = qsc::circuit::CircuitGroup {
+        circuits: vec![circuit],
+        version: Some(qsc::circuit::CURRENT_VERSION),
+    };
+
+    serde_wasm_bindgen::to_value(&circuitData).map_err(|e| e.to_string())
+}
+
```

Comment:
It feels to me like this should continue to return a bare `Circuit` and the wrapping of it in a `CircuitGroup` can be handled on a higher layer.